### ISSUE TITLE
Split multimedia to "music" and "video players"

### ISSUE THREAD

**Issue by epicEaston197** [Yellow-Dog-Man/Resonite-Issues#1800]
### Is your feature request related to a problem? Please describe.

When people have music playing in the world it is a really common occurrence for music to be on the same audio type as video players which makes it really annoying if you wanna watch a video while there's music in the world this results in you needing to find the volume slider in the world turn it local turn it down then you can watch your video with your friends this is really inconvenient and could just be done through the settings menu through a new media type called music

### Describe the solution you'd like

i would like multimedia to be Split into "music" and "video players"

### Describe alternatives you've considered

Maybe asking the world creator to put their music underneath sound effects but this comes with its own downsides also the default audio stream interface has it set to multimedia by default

### Requesters

@epicEaston197

**Comment by EmergencyTemporalShift** [comment]
I could get behind this. Even though some people play music through videos, this suggestion could add more granularity.

**Comment by epicEaston197** [comment]
> I could get behind this. Even though some people play music through videos, this suggestion could add more granularity.

Yeah that's also a concern of mine but the user could optionally edit the media type of the video player into a audio player that is already preconfigured to have music as its media type

**Comment by epicEaston197** [comment]
> Hmm-I feel like the issue you are highlighting could instead be better resolved if there were a mechanism/UI to find/list, and locally adjust the volume of/mute individual audio outputs. This could also be of benefit if a given audio output doesn't have a UI attached to it like a video player, or it can't otherwise be easily located, e.g. it's playing in broadcast.

This doesn't seem very useful when the list is just going to contain a list of non discreet unlabeled audio outputs this would likely take even longer to find then if I just went in the world and found the volume slider myself besides that every game I've ever played has had a music slider of some sort 

so it feels off for Resonite not to have one and again this is just an option for creators to categorize their audio and their worlds and items I don't see the downside of adding an extra volume slider for music

**Comment by EmergencyTemporalShift** [comment]
> This doesn't seem very useful when the list is just going to contain a list of non discreet unlabeled audio outputs this would likely take even longer to find then if I just went in the world and found the volume slider myself besides that every game I've ever played has had a music slider of some sort

What if you got a volume preview of each one, wouldn't that help narrow it down?

**Comment by epicEaston197** [comment]
> > This doesn't seem very useful when the list is just going to contain a list of non discreet unlabeled audio outputs this would likely take even longer to find then if I just went in the world and found the volume slider myself besides that every game I've ever played has had a music slider of some sort
> 
> What if you got a volume preview of each one, wouldn't that help narrow it down?

Maybe but it just doesn't seem like a very elegant solution to this problem even if there was volume previews the amount of audio outputs in the world could be countless with people spawning out videos it would probably take a while for you to find the music in the world but by then in that time you could have found the music player and just lower down the volume by then

it seems like an overly complex brute force solution for a really simple problem adding a music slider and a music media type would fix my issue

**Comment by art0007i** [comment]
I think having a music audio group would be good. I want multimedia to be up, but most of the time I have my own music playing and would prefer to keep music at 0. Currently I'd have to unmute multimedia, find the world background music and turn it off, just so I can watch one video that someone spawned out.
